build.sbt

```diff
@@ -14,40 +14,31 @@
  * limitations under the License.
  */
 
-val scala2_12 = "2.12.12"
-val scala2_13 = "2.13.8"
+ThisBuild / majorVersion := 0
+ThisBuild / scalaVersion := "2.13.12"
 
 val compile: Seq[ModuleID] = Seq(
-  "uk.gov.hmrc"       %% "domain"                            % "8.1.0-play-28",
-  "uk.gov.hmrc.mongo" %% "hmrc-mongo-work-item-repo-play-28" % "0.74.0",
-  "uk.gov.hmrc"       %% "http-verbs-play-28"                % "14.8.0",
-  "commons-codec"     %  "commons-codec"                     % "1.9"
-)
-
-// Play 28 (2.8.18) specific versions - caution when upgrading
-val play28: Seq[ModuleID] = Seq(
-  "com.typesafe.play" %% "play-json"            % "2.8.2",
-  "com.typesafe.play" %% "play-json-joda"       % "2.8.2",
-  "com.beachape"      %% "enumeratum"           % "1.7.0",
-  "com.beachape"      %% "enumeratum-play-json" % "1.7.0"
+  "uk.gov.hmrc"       %% "domain-play-30"                    % "9.0.0",
+  "uk.gov.hmrc.mongo" %% "hmrc-mongo-work-item-repo-play-30" % "1.7.0",
+  "uk.gov.hmrc"       %% "http-verbs-play-30"                % "14.12.0",
+  "commons-codec"     %  "commons-codec"                     % "1.16.0",
+  "org.playframework" %% "play-json"                         % "3.0.2",
+  "org.playframework" %% "play-json-joda"                    % "3.0.2",
```

Comment:
Initially I was getting some error when compiling because of transitive dependency, but didnt check it before raising the PR, will try removing it again and see.. Will update you.. @ardwalker thanks